pkg/broker/ingress/server_manager.go

```diff
@@ -24,6 +24,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"knative.dev/eventing/pkg/eventingtls"
 	"knative.dev/eventing/pkg/kncloudevents"
+	"knative.dev/eventing/pkg/reconciler/broker"
```

Comment:
This import has the side effect of starting a whole bunch of informers

Reply:
ahh. So better keeping it in the `pkg/broker|filter` package then I guess

Reply:
Doesn't that have the same side effect?

For example, broker/filter package might need an informer that the controlled doesn't need which would waste resources or require additional permissions

Reply:
Even if it's not needed or it works fine right now, it seems we're creating code coupling that is not really necessary and might break in the future, the secret name can be considered a contract (which it is even today) because changing the secret name is not as simple as changing the constant (we need at least to change the cert manager resources but also migration, etc would be necessary)